src/main/java/net/fabricmc/tinyremapper/api/TrClass.java

```diff
@@ -0,0 +1,29 @@
+package net.fabricmc.tinyremapper.api;
+
+import java.util.Collection;
+import java.util.List;
+
+public interface TrClass {
+	TrEnvironment getClasspath();
+
+	int getAccess();
+
+	String getName();
+
+	String getSuperName();
+
+	String getSignature();
+
+	List<String> getInterfaceList();
```

Comment:
getInterfaces

Reply:
I think it is also better to change the return type of this method to a Collection since the `getMembers` is Collection.